pkg/settings/setting.go

```diff
@@ -90,6 +90,7 @@ var (
 	UIPath                              = NewSetting("ui-path", "/usr/share/rancher/ui")
 	UIDashboardIndex                    = NewSetting("ui-dashboard-index", "https://releases.rancher.com/dashboard/latest/index.html")
 	UIDashboardPath                     = NewSetting("ui-dashboard-path", "/usr/share/rancher/ui-dashboard")
+	UIDashboardHarvesterLegacyPlugin    = NewSetting("ui-dashboard-harvester-legacy-plugin", "https://releases.rancher.com/harvester-ui/plugin/harvester-1.0.3-head/harvester-1.0.3-head.umd.min.js")
```

Comment:
Moving forward we are alphabetizing and adding code comments to all new or changed settings. Can you add to the section below (starting with AuthTokenMax) and add a code comment explaining what it does. If you want to move other related UI settings down there at the same time then feel free.

Reply:
Good point! Sorry. I've moved it down along with other UI settings. There were a few I wasn't so sure about, so there's room to improve